README.md

```diff
@@ -79,6 +81,40 @@ Example:
         result = myfunction(comm)
         assert result # or ....
 ```
+
+### MPITestFixture
+
+You cannot combine `MPITestFixture` with other pytest fixtures or decorators, what you can't with the `MPITest` decorator.
+
+Example: Parameter variation with `pytest.mark.parametrize`
+
+```python
+from runtests.mpi import MPITestFixture
+import pytest
+
+comm = MPITestFixture([1,2,3, 4,10], scope='function')
+
+@pytest.mark.parametrize("msg",["hello","world"])
+def test_y(msg, comm):
+    print(msg, comm.Get_rank())
+```
+
+Example: Parameter variation with `pytest.fixture`
+```bash
+from runtests.mpi import MPITestFixture
+import pytest
+
+comm = MPITestFixture([1,2,3,4], scope='function')
```

Comment:
Glad to know it works just like that. Did you know if it works with other scopes?

Reply:
Yes I put this fixture with scope "session" into the `conftest.py` file.

Reply:
That's a neat trick!